.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,24 @@
 ---
-name: 'Pull Request: New issue'
-about: Report and propose a fix for a (small) issue in Hazel
+name: 'Pull Request'
+about: Propose a fix implementation for a (small) issue in Hazel
 
 ---
 
-#### Describe the issue
+#### Describe the issue (if no issue has been made)
 A clear and concise description of what the issue is.
 A screenshot or copy of the error could be helpful as well.
 
-#### Expected behavior
-A clear and concise description of what you expected to happen.
-
-#### Proposed fix _(Make sure you've read [on how to contribute](https://github.com/TheCherno/Hazel/blob/master/.github/CONTRIBUTING.md) to Hazel)_
-A short description of what fix this is and how it fixed the issue you described.
-
 #### PR impact _(Make sure to add [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords))_
 List of related issues/PRs this will solve:
 
  Impact                  | Issue/PR
 ------------------------ | ------
-Other issues this solves | None or #number(s)
+Issues this solves       | None or #number(s)
 Other PRs this solves    | None or #number(s)
 
+#### Proposed fix _(Make sure you've read [on how to contribute](https://github.com/TheCherno/Hazel/blob/master/.github/CONTRIBUTING.md) to Hazel)_
+A short description of what fix this is and how it fixed the issue you described.
```

Comment:
A short description of what **a** fix this is and how it fixed the issue you described.

Reply:
Maybe rather something like
```suggestion
A short description of what this fix is and how it fixed the issue you described.
```